src/Native/Runtime/PalRedhawk.h

```diff
@@ -661,7 +661,7 @@ EventDataDescCreate(_Out_ EVENT_DATA_DESCRIPTOR * EventDataDescriptor, _In_opt_
 }
 #endif // _EVNTPROV_H_
 
-extern GCSystemInfo g_SystemInfo;
+extern GCSystemInfo g_RhSystemInfo;
```

Comment:
`gcenv.windows.cpp` defines this symbol as well, `gcenv.unix.cpp` doesn't, so to avoid nameclash and ensure this is available in all configurations i've kept it and renamed it. A few pieces in the runtime use this, they could probably be fixed to not depend on this struct.